<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wacom working area setter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>365</width>
     <height>252</height>
    </size>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>411</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select device</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dev_list"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="wid_btn">
         <property name="text">
          <string>Select Window</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>window id</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="wid_input"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>window size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dev_list_2">
         <item>
          <property name="text">
           <string>Select one</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1920x1080</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1080x1024</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="w_y"/>
       </item>
       <item>
        <widget class="QLineEdit" name="w_x"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="set_to_wnd_btn">
         <property name="text">
          <string>Set area to window</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="set_to_dsk_btn">
         <property name="text">
          <string>Set area to Desktop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>463</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
